src/diff/props.js

```diff
@@ -55,16 +55,15 @@ function setStyle(style, key, value) {
  * @param {boolean} isSvg Whether or not this DOM node is an SVG node or not
  */
 function setProperty(dom, name, value, oldValue, isSvg) {
-	if (isSvg) {
-		if (name === 'className') {
-			name = 'class';
-		}
-	} else if (name === 'class') {
+	if (name === 'key' || name === 'children') return;
```

Comment:
return is a non-minified keyword so this most of the time will introduce extra code compared to making one big if else branch